packages/appstore/src/modules/tour-guide/tour-guide.tsx

```diff
@@ -106,12 +125,11 @@ const TourGuide = () => {
                 trackOnboardingRestart();
                 setIsFirstTimeVisit(false);
                 toggleIsTourOpen(true);
+                setJoyrideIndex(0);
             }}
         />
     );
-
     const low_risk = content_flag === ContentFlag.LOW_RISK_CR_NON_EU || content_flag === ContentFlag.LOW_RISK_CR_EU;
-
     if (!is_tour_open && !should_trigger_tour_guide) return null;
```

Comment:
This is added to fix the overlay issue. If I remove this and add condition to run the overlay is not visible when clicking view tutorial multiple times